fvm/accounts_test.go

```diff
@@ -901,21 +847,18 @@ func TestRemoveAccountKey(t *testing.T) {
 		t.Run(fmt.Sprintf("Key added by a different api version %s", test.apiVersion),
 			newVMTest().withContextOptions(options...).
 				run(func(t *testing.T, vm *fvm.VirtualMachine, chain flow.Chain, ctx fvm.Context, view state.View, programs *programs.Programs) {
+					if test.apiVersion == accountKeyAPIVersionV2 {
+						t.Skip("Can no longer add keys with API v1")
+					}
+
```

Comment:
Just flagging: These tests might be needed for the cadence-m5, since there was a last-minute decision to revert back that change: https://github.com/onflow/cadence/pull/1604

SHouldn't be a problem for the current version (m4?) though

Reply:
Wow didn't know we are adding that back! Good to know. TY